psychopy/event.py

```diff
@@ -78,6 +89,55 @@
     # eventThread = _EventDispatchThread()
     # eventThread.start()
 
+    # GLFW keycodes for special characters
```

Comment:
This needs preceding with 
```
if haveGLFW:
```
otherwise people without glfw installed get a `NameError: name 'glfw' is not defined` here